go/stellar/stellar.go

```diff
@@ -21,6 +21,21 @@ import (
 	"github.com/stellar/go/amount"
 )
 
+// setNewWalletDisplayCurrency takes initial bundle that user creates
+// and sets primary account display currency based on OS region
+// settings (that part's TODO - right now it just sets "USD").
```

Comment:
This is a pretty far reaching TODO. I'd say just inline the function and if you want add the comment about what it really does to CreateWallet's comment. To implement that region detection deserves its own ticket.